apps/convergecfd/run_ccfd_3.0_impi.pbs

```diff
@@ -0,0 +1,91 @@
+#!/bin/bash
+
+#PBS -N ConvergeCFD
```

Comment:
As all the PBS options are specified thru the QSUB can these lines be removed and the script renamed from .pbs to .sh ?

Reply:
I would rather leave it as a .pbs file extention and leave in the #PBS lines since they are comments anyways and make it easier to submit the job when using PBS. Also the script calls out $PBS_O_WORKDIR, $PBS_NODEFILE, $PBS_HOSTFILE, and $PBS_JOBID so it would not work for another workload manager. If it needed to be run in SLURM or LSF then we would need to change those values in order to get it to work.